tests/common/auth_test.go

```diff
@@ -629,6 +629,99 @@ func TestAuthTestInvalidMgmt(t *testing.T) {
 	})
 }
 
+func TestAuthLeaseRevoke(t *testing.T) {
+	testRunner.BeforeTest(t)
+	ctx, cancel := context.WithTimeout(context.Background(), 30*time.Second)
+	defer cancel()
+	clus := testRunner.NewCluster(ctx, t, config.WithClusterConfig(config.ClusterConfig{ClusterSize: 1}))
+	defer clus.Close()
+	cc := testutils.MustClient(clus.Client())
+	testutils.ExecuteUntil(ctx, t, func() {
+		require.NoErrorf(t, setupAuth(cc, []authRole{testRole}, []authUser{rootUser, testUser}), "failed to enable auth")
+		rootAuthClient := testutils.MustClient(clus.Client(WithAuth(rootUserName, rootPassword)))
+
+		lresp, err := rootAuthClient.Grant(ctx, 10)
+		require.NoError(t, err)
+		err = rootAuthClient.Put(ctx, "key", "value", config.PutOptions{LeaseID: lresp.ID})
+		require.NoError(t, err)
+
+		_, err = rootAuthClient.Revoke(ctx, lresp.ID)
```

Comment:
If we have a test targeting this method specifically, it would be good to assert what's returning actually. 

[I know that it was not tested previously... ]

Reply:
clientv3 Lease revoke response is the type alias of the proto buf generated one. It only contains `ResponseHeader`. 

```
type LeaseRevokeResponse struct {
	Header               *ResponseHeader `protobuf:"bytes,1,opt,name=header,proto3" json:"header,omitempty"`
	XXX_NoUnkeyedLiteral struct{}        `json:"-"`
	XXX_unrecognized     []byte          `json:"-"`
	XXX_sizecache        int32           `json:"-"`
}
```

I was blindly migrate the old logic to this one. How about assert the revoked lease is no longer in LeaseList response and get response is empty?